application/pr/viewer.jsx

```diff
@@ -120,7 +120,7 @@ export default class PRViewer extends React.Component {
             if (st.pr.acceptable === "yes") {
                 let revert = "";
                 if (st.pr.markedAsNonSubstantiveBy) {
-                     revert = <span><button name="substantive" onClick={this.markSubstantiveOrNot.bind(this)}>Unmark as non-substantive</button></span>;
+                     revert = <span><button name="substantive" onClick={this.markSubstantiveOrNot.bind(this)}>Unmark as non-substantive</button><a href="https://www.w3.org/Consortium/Process/#substantive-change" title="definition of a substantive change in the W3C Process">🛈</a></span>;
```

Comment:
It seems that unicode character requires to install some other fonts. Maybe switching to ℹ could be more convenient.